apps/web-mzima-client/src/app/shared/components/support-modal/support-modal.component.ts

```diff
@@ -38,7 +38,7 @@ export class SupportModalComponent extends BaseComponent {
         title: this.translate.instant('app.documentation.title'),
         description: this.translate.instant('app.documentation.description'),
         action: () => {
-          this.openUrl('https://docs.ushahidi.com/ushahidi-platform-user-manual');
+          this.openUrl('https://docs.ushahidi.com/platform-user-manual/v/mzima');
```

Comment:
@MMwandigha since we have agreed on the call yesterday (with Cecilia) to make the mzima doc the default, you can just remove the `/v/mzima` at the end of the url.

Reply:
@Ifycode Noted. Let me adjust the link